lerobot/scripts/eval.py

```diff
@@ -453,6 +453,7 @@ def main(
         hydra_cfg = init_hydra_config(str(pretrained_policy_path / "config.yaml"), config_overrides)
     else:
         hydra_cfg = init_hydra_config(hydra_cfg_path, config_overrides)
+    logging.info(hydra_cfg)
```

Comment:
I'm not sure we want this on `eval.py`.

- It takes up a lot of lines on the terminal and when running multiple evals in a row (something I've done a lot in the past) you lose your place.
- Composition of multiple configs is not a problem here. You can just look at the config file in the pretrained model directory.
- Many of the params are relevant to training.

Reply:
OK, let's remove it from eval.py